cellprofiler/modules/measurecorrelation.py

```diff
@@ -83,6 +116,13 @@ def create_settings(self):
         self.spacer_2 = cps.Divider(line=True)
         
         self.add_object_button = cps.DoSomething("", 'Add another object', self.add_object)
+        
+        self.thr = cps.Float(
+            "Set threshold as percentage of maximum intensity for the images",
+            15, minval=0, maxval=99, doc='''\
+            Select the threshold as a percentage of the maximum intensity of the above image [0-99].''')
```

Comment:
If a setting is added to a module, the following changes need to be made to ensure backward compatibility with old pipelines:

- The `variable_revision_number` needs to be incremented to distinguish the old settings from the new.
- The module's `upgrade_settings` method needs to be modified to convert the settings format of the old variable revision into the new.

At the same time, we like to add a test to the module's test suite to check that a pipeline with the new settings format can be loaded.